src/utils/type-transform/String-ArrayBuffer.ts

```diff
@@ -1,19 +1 @@
-export function encodeText(str: string) {
```

Comment:
Remove this file and import those functions from the package directly.

Reply:
minimize changes